src/node/node_state.h

```diff
@@ -1372,6 +1377,7 @@ namespace ccf
 
         GenesisGenerator g(network, tx);
         g.open_service();
+        trigger_snapshot(tx);
```

Comment:
I'm guessing this is the perf hit. This adds a snapshot on every service open, and the perf tests probably didn't produce any snapshot previously.

Reply:
Yeah, that's a good point! I would have expected that impact to be less severe, but you may be right.